corehq/apps/data_interfaces/utils.py

```diff
@@ -34,3 +35,33 @@ def add_cases_to_case_group(domain, case_group_id, uploaded_data):
         case_group.save()
 
     return response
+
+
+def archive_forms(user, uploaded_data):
```

Comment:
i think it should go in the view so it can provide immediate feedback to the user. only thing is i didn't really expect that use case and i have a `list` call on the uploaded data that isn't done on a separate thread. i imagine that could take an extremely long time if there were 100,000 docs. i think one option could be to limit the size of the file which could be a much quicker operation to do before opening and reading the rows. if it's too big throw an exception before doing anything else. what do you think?